sites/all/modules/hr/hr_datasets/js/hr_datasets.js

```diff
@@ -142,12 +147,36 @@ Drupal.behaviors.hrDatasets = {
         },
 
         searchByTitle: function(event) {
-          var val = $('#search').val();
+          if(event.keyCode == 13) {
```

Comment:
I think we need to use which as well as keycode then.
Does this work well on mobile?